src/sections/dataset/dataset-action-buttons/edit-dataset-menu/DeaccessionDatasetButton.tsx

```diff
@@ -48,13 +48,16 @@ export function DeaccessionDatasetButton({
   } = useForm<DeaccessionFormData>({
     defaultValues: { versions: defaultVersions, deaccessionForwardUrl: '' }
   })
+
   if (
+    dataset.version.publishingStatus === DatasetPublishingStatus.DEACCESSIONED ||
+    dataset.version.publishingStatus === DatasetPublishingStatus.DRAFT ||
     !dataset.version.someDatasetVersionHasBeenReleased ||
```

Comment:
```suggestion
  dataset.version.publishingStatus !== DatasetPublishingStatus.RELEASED ||
```

Reply:
I think it's ok to just check if the DatasetPublishingStatus for this version !== RELEASED, since it covers the other version states.